Ot285 127 layout backoffice

nahuelba opened the pull request:
- Añadido backoffice component
- Añadida funcionalidad de ocultar/mostrar sidebar con header del backoffice
- Modificación css de top en sidebar, y agregado router outlet dentro de mat-sidenav-content
- Routeo del backoffice

nahuelba committed:
backoffice added, sidebar modified

nahuelba committed:
Merge branch 'main' into OT285-127-layout-backoffice

nahuelba committed:
sidebar functionality added

nahuelba committed:
return type

nahuelba committed:
removed unused css

nahuelba committed:
removed more unused css

carolinalaborde commented on src/app/core/components/backoffice/backoffice.component.html:
```diff
@@ -0,0 +1,5 @@
+<app-header (OpenCloseSidebar)="sidebar = !sidebar"></app-header>
+
+
+ <app-sidebar [opened]="sidebar"></app-sidebar> 
```
Falta router-outlet debajo

carolinalaborde commented on src/app/features/app-routing.module.ts:
```diff
@@ -50,6 +51,7 @@ const routes: Routes = [
 
   {
     path: "backoffice",
+    component:BackofficeComponent,
```
El layout del backoffice no debe ir en una ruta

nahuelba committed:
Change name layout backoffice
router-outlet and ng-content added
Routing removed
app-layout-backoffice in app.component added

nahuelba committed:
removed header routes
Sidebar backoffice route name changed to  Dashboard

nahuelba committed:
removed router-outlet from app.component
